test/e2e/shared_publisher_clients.test.ts

```diff
@@ -64,9 +65,12 @@ describe("publish messages through multiple publishers", () => {
       }
     }
 
-    await client.declareConsumer({ stream: streamName, offset: Offset.first() }, (msg) => {
-      received.push(msg.content.toString("utf-8"))
-    })
+    await client.declareConsumer(
+      { stream: streamName, offset: Offset.first(), creditPolicy: creditsOnChunkReceived(1, 1) },
```

Comment:
Same as other comment, is the non-default creditPolicy important here? Should that be documented? Or is it only for performance reasons?